packages/expo-ads-admob/android/src/main/java/expo/modules/ads/admob/AdMobRewardedVideoAdModule.java

```diff
@@ -55,7 +46,8 @@ public enum Event {
     }
 
     @Override
-    public @NonNull String toString() {
+    public @NonNull
+    String toString() {
```

Comment:
This order looks weird 😅 
```suggestion
    @NonNull
    public String toString() {
```